flink-tests/src/test/java/org/apache/flink/test/streaming/experimental/CollectITCase.java

```diff
@@ -7,7 +7,7 @@
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
  *
- *     http://www.apache.org/licenses/LICENSE-2.0
+ * http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
There are a bunch of these types of formatting changes. They actually make the license header different from the other files. Would be good to revert this.